packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -98,15 +107,26 @@ function _domBreadcrumb(dom: BreadcrumbsOptions['dom']): (handlerData: { [key: s
     let target;
     let keyAttrs = typeof dom === 'object' ? dom.serializeAttribute : undefined;
 
+    let maxStringLength =
+      typeof dom === 'object' && typeof dom.maxStringLength === 'number' ? dom.maxStringLength : undefined;
+    if (maxStringLength && maxStringLength > MAX_ALLOWED_STRING_LENGTH) {
+      if (__DEBUG_BUILD__) {
```

Comment:
Could you re-write this to:

```
__DEBUG_BUILD__ && logger.warn(...);
```

Just for consistency with the rest of the codebase :)